lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -16,28 +16,42 @@ def identifier
     end
 
     def document
+      {}.merge(common_fields)
+        .merge(fields_from_expanded_links)
```

Comment:
I know we seem to like manually creating the JSON output at GDS, have we looked at/thought about using a gem to do the combinational work, this leaves more easily testable methods, rather than only being able to test the JSON response?

i.e. something like ActiveModel::Serializer